packages/react/src/components/LocaleModal/LocaleModalCountries.js

```diff
@@ -208,6 +212,18 @@ export const filterLocale = (
     localeItems.length === localeItemsHidden.length
       ? modalLabels.unavailabilityText
       : modalLabels.availabilityText;
+
+  /**
+   *  Reflect number of results in the accessibility readout
+   */
+  const resultsElement = document.querySelector(`.${prefix}--assistive-text`);
+  const resultsCount = localeItems.length - localeItemsHidden.length;
+
+  if (resultsElement) {
+    resultsElement.innerHTML = `${resultsCount} ${
```

Comment:
`innerHTML` causes HTML parser to run and thus using it for text content will be redundant:

```suggestion
    resultsElement.textContent = `${resultsCount} ${
```